opcodes/arc-tbl.h

```diff
@@ -8043,7 +8046,10 @@
 { "jl", 0x20E20F80, 0xF8FF0FE0, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, NONE, { LIMM }, { C_CC }},
 
 /* jli_s u10 010110uuuuuuuuuu.  */
-{ "jli_s", 0x00005800, 0x0000FC00, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, CD1, { UIMM10_6_S }, { 0 }},
+{ "jli_s", 0x00005800, 0x0000FC00, ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, JUMP, ARC_CD, { UIMM10_6_S }, { 0 }},
```

Comment:
Hmm, probably because we may have an issue with the gcc driver. But this should not be the reason for this mod. I've checked the jli_s instruction with the assembler only, and it works correctly. And, by all means, ARC_CD is wrong, when you are not sure about the instruction SUBCLASS, we should use NONE.